tests/testthat/data-citation/encoding/inst/CITATION

```diff
@@ -0,0 +1,14 @@
+citHeader("To cite bigsnpr in publications, please use:")
+
+citEntry(entry = "Article",
+         title="Efficient management and analysis of large-scale genome-wide data with two R packages: bigstatsr and bigsnpr",
+         author=c("Florian Privé", "Hugues Aschard", "Michael G.B. Blum"),
```

Comment:
Note that this CITATION file has an author with non-ASCII (Prive) and calls citation(auto = meta)